### PR TITLE
{diskpool} Refine output and add new command

### PR DESCRIPTION
**Pull request by Juliehzl**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
diskpool

File: src/diskpool/azext_diskpool/generated/_help.py [hunk 1]
```diff
@@ -98,6 +98,16 @@
                az disk-pool delete --name "myDiskPool" --resource-group "myResourceGroup"
 """
 
+helps['disk-pool list-outbound-network-dependency-endpoint'] = """
+    type: command
+    short-summary: "Gets the network endpoints of all outbound dependencies of a Disk Pool."
```
Comment (zhoxing-ms) [inline src/diskpool/azext_diskpool/generated/_help.py:103]:
```suggestion
    short-summary: Get the network endpoints of all outbound dependencies of a Disk Pool.
```

File: src/diskpool/azext_diskpool/manual/_transformers.py [hunk 2]
```diff
@@ -35,8 +37,56 @@ def transform_disk_pool_list_output(result):
     usable by the CLI and tools such as jpterm. """
     return build_table_output(result, [
         ('Name', 'name'),
-        ('Availability Zones', 'availabilityZones'),
+        ('Resource Group', 'resourceGroup'),
```
Comment (zhoxing-ms) [inline src/diskpool/azext_diskpool/manual/_transformers.py:40]: May I ask why do we need to modify those value?

File: src/diskpool/azext_diskpool/generated/_help.py (hunk 1 above)
Comment (Juliehzl) [reply src/diskpool/azext_diskpool/generated/_help.py:103]: this is generated code

File: src/diskpool/azext_diskpool/manual/_transformers.py (hunk 2 above)
Comment (Juliehzl) [reply src/diskpool/azext_diskpool/manual/_transformers.py:40]: as service request

**Review by zhoxing-ms — APPROVED** [review]

File: src/diskpool/azext_diskpool/manual/_transformers.py (hunk 2 above)
Comment (zhoxing-ms) [reply src/diskpool/azext_diskpool/manual/_transformers.py:40]: OK

**Pull request merged**